Rakefile

```diff
@@ -27,6 +27,11 @@ RSpec::Core::RakeTask.new(:acceptance) do |t|
     t.pattern = 'spec/acceptance'
 end
 
+desc "Run integration tests"
+task :integration do
+  sh("BEAKER_setfile=spec/acceptance/beaker/nodesets/rhel7.yaml bundle exec rspec spec/acceptance/integration")
```

Comment:
This would be better to use the built-in rake task so as to allow overriding the individual specs and the set as needed using the standard environment variables. ie.

```ruby
RSpec::Core::RakeTask.new(:acceptance) do |t|
  t.pattern = 'spec/acceptance/integration'
end
```